fuzzel-emoji

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+wtype 0
```

Comment:
Hmm, found this was inserting 0 and there's a PR in wofi-emoji to fix: https://github.com/dln/wofi-emoji/pull/10/files - will update the PR